src/Dialogs/Message.cpp

```diff
@@ -153,3 +153,77 @@ ShowMessageBox(const TCHAR *text, const TCHAR *caption, unsigned flags)
 
   return res;
 }
+
+
+
+int    //////// LOCKSCREEN
+ShowLockBox()
+{
+
+  SingleWindow &main_window = UIGlobals::GetMainWindow();
+
+  const unsigned button_width = Layout::Scale(15u);
```

Comment:
`Layout::Scale()` is a bad metric for this. What you probably want is `GetMinimumControlHeight()` or `GetMaximumControlHeight()`.

Reply:
Why did you mark this resolved? You didn't change this line. It still uses `Layout::Scale()`.

Reply:
I just copied that part from the original ShowMessageBox function...
After your suggestion I changed the height row to use GetMinimumControlHeight but I couldn't find any similar function for width.

Reply:
Changed it to make a square button, but not in a way you'll like, I guess :-P